srcpkgs/freeciv/template

```diff
@@ -10,12 +10,14 @@ makedepends="SDL2_gfx-devel SDL2_image-devel SDL2_mixer-devel SDL2_ttf-devel
 short_desc="Free and Open Source empire-building strategy game - server/data"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later"
-homepage="http://freeciv.org/"
-changelog="https://freeciv.fandom.com/wiki/NEWS-${version}"
+homepage="https://www.freeciv.org/"
+changelog="https://www.freeciv.org/wiki/NEWS-${version}"
 distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-${version}.tar.xz"
 checksum=13215adc96be9f2894d5f3a12c78b8ebb9ae06ecdab25fe6bb1794f6e6d2b61b
 nopie_files="/usr/bin/freeciv-ruledit"
 
+# TODO: gtk3 should be replaced by better supported gtk3.22 in any
+#       distribution that has gtk+-3.22 or gtk+-3.24 available.
```

Comment:
Why?

We have `gtk+3-3.24.34_1`, what is the purpose of this comment?

Reply:
Freeciv gtk3-client is an older client that requires only gtk+-3.10. Freeciv gtk3.22-client is better supported client that requires gtk+-3.22 or gtk+-3.24. Because you have that gtk+-3.24.34, and have no need for the compatibility with gtk+ < 3.22, you should switch to gtk3.22-client of Freeciv.

Reply:
Ah, so you are proposing

```
-_clients="gtk3,sdl2"
+_clients="gtk3.22,sdl2"
```
?

Reply:
Yes. Likely requires more changes than that to work properly, but that's the core of it.